nuitka/freezer/DependsExe.py

```diff
@@ -118,14 +132,44 @@ def _parseDependsExeOutput2(lines):
         if "E" in line[: line.find("]")]:
             continue
 
-        # Skip missing DLLs, apparently not needed anyway, but we can still
-        # try a few tricks
+        # Try to find missing DLLs for PythonXY.dll and MSVC Redist dlls.
+        VCREDIST_DLLS = {
+            "concrt140.dll",
+            "msvcp140.dll",
+            "msvcp140_1.dll",
+            "msvcp140_2.dll",
+            "msvcp140_atomic_wait.dll",
+            "msvcp140_codecvt_ids.dll",
+            "vccorlib140.dll",
+            "vcruntime140.dll",
+            "vcruntime140_1.dll",
+            "vcruntime140_threads.dll",
+            "vcamp140.dll",
+            "vcomp140.dll",
+        }
+
         if "?" in line[: line.find("]")]:
-            # One exception are "PythonXY.DLL", we try to find them from Windows folder.
-            if dll_filename.startswith("python") and dll_filename.endswith(".dll"):
-                dll_filename = _attemptToFindNotFoundDLL(dll_filename)
+            is_vcredist_dll = os.path.basename(dll_filename) in VCREDIST_DLLS
+            is_python_dll = dll_filename.startswith("python") and dll_filename.endswith(
```

Comment:
Just noticing that "python" is not treated the same, i.e. no basename taken. Apparently as a DLL it's usually found either way.